aws/onboarding/byok8s/manual-onboarding-steps/README.md

```diff
@@ -0,0 +1,206 @@
+# Documentation for Manually Creating an EKS Cluster for RSC Exocompute
```

Comment:
Let's rename to `manual-steps`. It is already under `onboarding` and even for python.

Reply:
Updated